### PR TITLE
[el] add greek translation

### PR DESCRIPTION
**Pull request by sourikata2**
**Description**
I added the el translations suite.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [x] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "ext-tokenizer": "*",
         "codeigniter/coding-standard": "^1.1",
         "codeigniter4/codeigniter4": "4.x-dev",
-        "friendsofphp/php-cs-fixer": "^3.1",
+        "friendsofphp/php-cs-fixer": "^3.40",
```
Comment (kenjis) [inline composer.json:15]: Why do you change this?

Comment (sourikata2) [reply composer.json:15]: I made composer require in order to run the php-cs-fixer. Was it  a mistake?

Comment (kenjis) [reply composer.json:15]: Yes, you should have use `composer update` instead of `composer require`.
Can you remove the change?

Comment (sourikata2) [reply composer.json]: I removed the change.
Please can you guide me in the failed tests on how to resolve them?

**Comment by kenjis** [comment]
Okay, all checks have passed!

**Review by kenjis — APPROVED** [review]

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**